app.js

```diff
@@ -0,0 +1,10 @@
+const open = document.getElementById('btnOpen');
+const close = document.getElementById('btnClose');
+
+open.addEventListener('click', () => {
+  close.style.display = 'block';
+});
+
+close.addEventListener('click', () => {
+  close.style.display = 'none';
+});
```

Comment:
- [ ] Good job by implementing the menu but you should also make sure your design stick as much as possible to the Figma design (eg: color, margin,...) 🙃 😉 

Your design
<img width="326" alt="Screen Shot 2023-01-31 at 23 32 25" src="https://user-images.githubusercontent.com/74501523/215881988-a0cbab5e-5eb8-4370-8e8c-cb98c9b8db5e.png">

Figma design
<img width="276" alt="Screen Shot 2023-01-31 at 23 33 34" src="https://user-images.githubusercontent.com/74501523/215882044-6442a0ee-8d46-4857-a7bc-9905aef8944c.png">

- [OPTIONAL] Also I noticed that in the mobile menu when I clicked (or tapped) any option on the menu, the menu disappear. But is supposed to be just for the mobile menu and not for the desktop menu, the menu should not disappear when the user clicks or tap any option in the menu. Please kindly fix that 🙃 🙃

Reply:
Hello Cynthia! Nice to meet you! Thanks for the feedback, I just want to let you know that the color is what figma provides you in their template, I took the color code from there. I also kindly want to suggest that a little extra space in the design doesn't seem like enough reason for me to not approve my work, since the instructions are clear about it, "as much as possible"... There is not always something to say, sometimes the works can pass on the first try, that's what the optional suggestions are for. But thanks I really appreciate your feedback, Regards Cynthia!

Reply:
Hi @Jrmcuervo,
So first the design is a part of the requirement, and making sure to follow exactly the design can help you in the future in case you have a client that gives you a design to reproduce, you should pay attention even to every small detail and make sure to satisfy your client also that makes you more professional and confident.
Secondly, you don't use the same color code as in the figma design. The color code for figma is `#6070ff` but you used `#5564ebb5`, you can even see in the two screenshots the color is not the same.
But if you are not okay with my review you can request a second opinion 🙏